### ISSUE TITLE
Load Land Bank Parcels using FDW

### ISSUE THREAD

**Issue by zmon** [codeforkansascity/address-api#59]
https://maps.kcmo.org/kcgis/rest/services/external/DataLayers/MapServer/12/query

curl  --data 'geometry=&geometryType=esriGeometryPolygon&inSR=4326&spatialRel=esriSpatialRelIntersects&relationParam=&objectIds=&where=1%3D1&time=&returnCountOnly=false&returnIdsOnly=false&returnGeometry=true&maxAllowableOffset=&outSR=&outFields=&f=pjson' https://maps.kcmo.org/kcgis/rest/services/external/DataLayers/MapServer/12/query > p.json

ogr_fdw_info -s p.json
ogr_fdw_info -s p.json -l OGRGeoJSON

CREATE SERVER myserver
  FOREIGN DATA WRAPPER ogr_fdw
  OPTIONS (
    datasource 'p.json',
    format 'GeoJSON' );

CREATE FOREIGN TABLE ogrgeojson (
  fid integer,
  geom geometry,
  objectid integer,
  kivapin varchar,
  landusecode integer,
  apn varchar,
  address varchar,
  addr integer,
  fraction varchar,
  prefix varchar,
  street varchar,
  street_type varchar,
  suite varchar,
  own_name varchar,
  own_addr varchar,
  own_city varchar,
  own_state varchar,
  own_zip varchar,
  shape_area real,
  shape_len real )
  SERVER myserver
  OPTIONS ( layer 'OGRGeoJSON' );

select kivapin, landusecode, apn, address, own_name from ogrgeojson;

| JSON Field | Sample Value | Our use |
| --- | --- | --- |
| OBJECTID | 567609 | NA |
| KIVAPIN | 22903 | Use for primary parcel lookup |
| LANDUSECODE | 1111 | Data to capture for parcel |
| APN | JA28830051200000000 | Use for secondary parcel lookup |
| ADDRESS | 3129 Elmwood Ave | Use for tertiary parcel lookup |
| ADDR | 3129 | NA |
| FRACTION | null | NA |
| PREFIX | null | NA |
| STREET | Elmwood | NA |
| STREET_TYPE | Ave | NA |
| SUITE | null | NA |
| OWN_NAME | Land Bank of Kansas City Missouri | Update |
| OWN_ADDR | 4900 Swope Pkwy 2nd Fl | Update |
| OWN_CITY | Kansas City | Update |
| OWN_STATE | MO | Update |
| OWN_ZIP | 64130 | Update |
| SHAPE.AREA | 4243.0264236111107 | NA ?? Ron |
| SHAPE.LEN | 313.01321730978202 | NA ?? Ron |
| geometry | Jason Data | Use for quaternary parcel lookup?? |

```
{
      "attributes" : {
        "OBJECTID" : 567609,
        "KIVAPIN" : "22903",
        "LANDUSECODE" : 1111,
        "APN" : "JA28830051200000000",
        "ADDRESS" : "3129 Elmwood Ave",
        "ADDR" : 3129,
        "FRACTION" : null,
        "PREFIX" : null,
        "STREET" : "Elmwood",
        "STREET_TYPE" : "Ave",
        "SUITE" : null,
        "OWN_NAME" : "Land Bank of Kansas City Missouri",
        "OWN_ADDR" : "4900 Swope Pkwy 2nd Fl",
        "OWN_CITY" : "Kansas City",
        "OWN_STATE" : "MO",
        "OWN_ZIP" : "64130",
        "SHAPE.AREA" : 4243.0264236111107,
        "SHAPE.LEN" : 313.01321730978202
      },
      "geometry" : {
        "rings" : [
          [
            [
              2780243.9383336324,
              1056426.5800002962
            ],
            [
              2780245.246666966,
              1056461.4300002977
            ],
            [
              2780246.3466669675,
              1056461.3883336335
            ],
            [
              2780365.6966669653,
              1056456.7133336291
            ],
            [
              2780366.7966669668,
              1056456.6716669649
            ],
            [
              2780365.4383336324,
              1056421.7966669649
            ],
            [
              2780243.9383336324,
              1056426.5800002962
            ]
          ]
        ]
      }
    },
```

**Comment by zmon** [comment]
See /data/KCMO-land-use-codes

[Table](https://github.com/codeforkansascity/kc-open-data-library/wiki)  that has descriptions for Land Use Codes  https://github.com/codeforkansascity/kc-open-data-library/wiki

[Data Only from Socrata](https://data.kcmo.org/resource/mgwp-vfsh.json) has two fields
- land_use_code
- land_use_description

Create Table

```
\c address_api

DROP TABLE IF EXISTS land_use_codes ;
DROP SEQUENCE IF EXISTS land_use_codes_id_seq;



CREATE SEQUENCE land_use_codes_id_seq
    START WITH 2001
    INCREMENT BY 1
    NO MINVALUE
    NO MAXVALUE
    CACHE 1;


ALTER TABLE public.land_use_codes_id_seq OWNER TO c4kc;

SET default_tablespace = '';

SET default_with_oids = false;

CREATE TABLE land_use_codes (
      id integer DEFAULT nextval('land_use_codes_id_seq'::regclass) NOT NULL,
      land_use_code character varying(10) DEFAULT NULL::character varying,
      land_use_description character varying(40) DEFAULT NULL::character varying,
      active integer DEFAULT 1,
      added timestamp without time zone DEFAULT now(),
      changed timestamp without time zone DEFAULT now()
);  

ALTER TABLE public.land_use_codes OWNER TO c4kc;
```

**Comment by zmon** [comment]
Closed with eae57c6
